test/functional/acl.test.js

```diff
@@ -268,7 +280,7 @@ describe('HTTP API access control', () => {
           participantId: 1,
           presence: 3,
           timestamp: new Date(),
-          authorId: 1,
+          authorId: otherUserId,
```

Comment:
The value of `otherUserId` isn't `1`, it's `123`. I just wanted to make sure that you were aware of this. it doesn't seem to cause any breakages in the tests but making these kinds of changes to the test data probably warrant a mention in the PR or commit message

Reply:
Yeah, the main point here is to have an `user.id` that does exist when the presence history row as otherwise the foreign key on `presence_history.authorId` will not allow creating the presence_history row